### PR TITLE
fix(macro): Apply usd_to_cad macro to all fields

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

File: server/macros/currency_usd_to_cad.py [hunk 1]
```diff
@@ -37,7 +37,22 @@ def convert(match):
         cad = rate * usd
         return 'CAD %d' % cad
 
-    item['body_html'] = re.sub('\$([0-9]+)', convert, item['body_html'])
+    # replacements
+    if item.get('body_html', None):
```
Comment (petrjasek) [inline server/macros/currency_usd_to_cad.py]: this could use some `for field in fields: ...`

**Pull request merged**